monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Challenges_With_Open_Software_Revieww.md

```diff
@@ -0,0 +1,68 @@
+
+Any questions? I think I might have like one minute for one question.
+
+Actually, we have a little bit...yeah, does anybody have any quick questions for Paul before we get going? We have at least one over here, wonderful.
+
+Question: You mentioned IRC and #monero-dev, of course. Is there, I mean...given the scope of the project IRC itself, due to the temporal nature, might not be the best for potentially planning and tracking over progress? Is there an issue or--I don't know--like an EPIC or something that we can potentially watch and chime into?
+
+Answer: So part of the problem with that is that we could call out specific areas of the code that need attention. But they're going to change relatively quickly. So I think it needs to be a living effort, and we need a more flexible approach. I do think that issue tracking is applicable. I think it's a good idea, and it'll be useful. But the people who are involved in that are going to have to be updating that stuff. And in some cases, it's just the case that it's more efficient for whoever notices the issue to just fix it right there, because, honestly, that's usually all it takes. And it's usually just the people who are like, "No, I'm not going to do that," like, "I have more important stuff to do." But actually, they don't, necessarily. I think that we might need to change our priorities a little bit in terms of how we write code. It's like we should always be factoring. ABF, always be factoring.
+
+We actually have more time for questions, if anyone can think of anything else. Okay, we have two over there at least.
+
+Question: So I wonder, as a follow on to that last question, I wonder whether it might be nice to have a dedicated IRC channel, actually, for this because I think one of the problems with new changes that come along, they get mixed up in some conversation, the current kind of things that they worked on as opposed to that taking a step to the side. And of course, many will still contribute into that channel, as well. But it's kind of like a different mindset, because what you're talking about here is really about a mindset change, I think, as much as it is, so, "Yeah, let's have a new IRC channel" and work on that.
+
+Answer: Yeah, so I actually made one, #monero-newlibs, but it's actually because of the fact that this needs to be integrated into our culture in a way. It started to occur to me that maybe it shouldn't be separate, at least not right now. There wasn't enough community awareness at the time, I think, to really make that an effective strategy. But I think that as this-- I mean, more and more people are starting to become involved in this. So I think it could actually make sense. I think that's a good idea. So that's #monero-newlibs. 
```

Comment:
Yeah, so I actually made one, #monero-newlibs, but it's actually because of the fact that this needs to be integrated into our culture in a way. It started to occur to me that maybe it shouldn't be separate, at least not right now. There wasn't enough community awareness at the time, I think, to really make that an effective strategy. But I think that as this-- I mean, more and more people are starting to become involved in this. So I think it could actually make sense. I think that's a good idea. So that's #monero-newlibs. 

**Remove the word "Answer"**